### ISSUE TITLE
/vocabulary/term/{term} not searching in Synonyms

### ISSUE THREAD

**Issue by Kabenla** [SciGraph/SciGraph#164]
It looks like the /vocabulary/term/{term} endpoint is not looking through synonyms.  
In the following example, I have loaded the SNOMED-CT  ontology into my scigraph installation.

Searching for the term "Group A2B" yields no results when using the /vocabulary/term/{term}
https://monosnap.com/file/1jXPx1PrCsFhvmnkM8K9YdIIxP8foZ

However "Group A2B" is a synonym of Blood group A>2<B (finding) which returns a node with the same endpoint 
https://monosnap.com/file/5ldhWkSlGqdfrrgtojcXkkecZs3nOO

For some reason the endpoint does not find the node when the parameter is "Group A2B" even though we set the searchSynonyms option to true.  

Have misunderstood what the searchSynonym option does?

**Comment by tgbugs** [comment]
Have you defined SNOMED-CT as a curie in the scigraph graph load yaml file? I can produce this behavior by specifying an invalid curie in the prefix field. Try removing the prefix and see if you get the expected result.

**Comment by Kabenla** [comment]
Yes SNOMED-CT has been mapped to http://snomed.info/id/ as shown here
https://monosnap.com/file/1BFxIV8pKZozUAsUgBK7qapsyMQyDb.  I removed the
curie from the search in Scigraph but still got no results for Group A2B in
/vocabulary/term/{term}

**Comment by tgbugs** [comment]
Hrm, another possibility: is the annotation property that SNOMED uses for synonyms mapped to synonym in the mappedProperties: section of the yaml file? e.g.
```
mappedProperties:
    - name: synonym
      properties:
        - http://snomed.info/id/snomed_synonym_iri_fragment
```

**Comment by Kabenla** [comment]
No, I have not put in that mapping for synonyms.  I will try it to see what
happens.  But if that is the issue, then how does Group A2B  appear as a
synonym?
